slices/gcc-14-aarch64-linux-gnu.yaml

```
@@ -0,0 +1,29 @@
+package: gcc-14-aarch64-linux-gnu
+
+essential:
+  - gcc-14-aarch64-linux-gnu_copyright
+
+slices:
+  # NOTE: You might want to create a symlink from /usr/bin/gcc to
+  #       /usr/bin/aarch64-linux-gnu-gcc to use this slice.
+  #       ln -s aarch64-linux-gnu-gcc-14 "$rootfs/usr/bin/gcc"
+  gcc-14:
+    essential:
+      - libc6_libs
+      - libgcc-14-dev_headers
```

```suggestion
    essential:
      - binutils-aarch64-linux-gnu_assembler
      - binutils-aarch64-linux-gnu_linker
      - libc6_libs
      - libgcc-14-dev_headers
```

We should include the `binutils`, right? Otherwise, the assembly and linking won't happen automatically for `gcc`.